lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-max-width: 300px;
+$nx-sidebar-width-open: 250px;
+$nx-sidebar-width-closed: 72px;
+$nx-sidebar-logo-height: 34px;
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-hover-color: $nx-indigo-600;
+
+$nx-sidebar-focus-border: 1px solid $nx-focus-border-color;
+
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar-operable {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  box-sizing: border-box;
+  color: $nx-sidebar-text-color;
+  max-width: $nx-sidebar-max-width;
+  padding: $nx-spacing-xl $nx-spacing-l $nx-spacing-l $nx-spacing-l;
+  width: $nx-sidebar-width-open;
+
+  .nx-text-link {
+    color: $nx-sidebar-text-color;
+  }
+}
+
+.nx-page-sidebar-operable__header {
+  display: flex;
+  margin-bottom: $nx-spacing-xl;
+}
+
+.nx-page-sidebar-operable__toggle {
+  border-color: transparent;
+  border-radius: 50%;
+  color: $nx-sidebar-text-color;
+  flex: none;
+  height: 34px;
```

Comment:
I thought this was supposed to be 32x32?

Reply:
> I thought this was supposed to be 32x32?
Here's the issue I'm having with `32px`:

When the sidebar is open the logo makes the header section 34 pixels in height; when the sidebar is closed there is no logo to maintain the 34 pixels in height and the whole header ends up being as tall as the toggle button, which causes a shift in the whole content by 2 pixels. Hence why I went with 34pixels instead.

Reply:
> When the sidebar is open the logo makes the header section 34 pixels in height; when the sidebar is closed there is no logo to maintain the 34 pixels in height and the whole header ends up being as tall as the toggle button, which causes a shift in the whole content by 2 pixels. Hence why I went with 34pixels instead.

I can think of two other solutions that don't involve deviating from the mockups:
* Putting a 34px height on the `.nx-page-sidebar-operable__header`
* Putting a 1px top and bottom margin on the button.

Reply:
> Putting a 34px height on the `.nx-page-sidebar-operable__header`

Done in d61a17e1d86bcd68b3db99a7a9aa097772981b38

Reply:
> Done in d61a17e

Probably want `align-items: center` on the `.nx-global-sidebar__header` so that the button is in the middle of that space

Reply:
> > Done in [d61a17e](https://github.com/sonatype/sonatype-react-shared-components/commit/d61a17e1d86bcd68b3db99a7a9aa097772981b38)
> 
> Probably want `align-items: center` on the `.nx-global-sidebar__header` so that the button is in the middle of that space

Added in c1bfaedff9f4c5d568cae5a4d606146ff5cb70cf